llvm/test/Transforms/Float2Int/pr79158.ll

```diff
@@ -1,5 +1,8 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
```

Comment:
the file name and function names should be more descriptive, but since it's pre-existing that can be a followup PR